### ISSUE TITLE
SSH key handling gets confused when two types of ssh keys are in use

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1054]
Originally reported of Google Code with ID 998.

```
<b>What steps will reproduce the problem?</b>
1. install a few nodes and equip them with more than one type of ssh key, for example id_dsa.pub and id_rsa.pub (and their private counterparts)
2. create a cluster, add the nodes

<b>What is the expected output? What do you see instead?</b>

Not all or not the right keys get added to the cluster's public key file.
```
Originally added on 2014-12-02 14:55:13 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed with:

commit 72ec4d81cd726465a17449f00a0f85735481bcff
Author: Helga Velroyen <helgav@google.com>
Date:   Fri Jan 9 10:52:28 2015 +0100

    Use DSA SSH keys only
    
    Ganeti always generates DSA SSH keys and thus, these should
    be the only once added to the public key file and the
    authorized_keys files. This fixes Issue 998.
    
    Signed-off-by: Helga Velroyen <helgav@google.com>
    Reviewed-by: Klaus Aehlig <aehlig@google.com>
```
Originally added on 2015-01-13 10:30:12 +0000 UTC.

Mentioned Issue with GoogleCode ID 998 has been migrated to GitHub with ID #1054.
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.14.0
```
Originally added on 2015-06-02 10:37:03 +0000 UTC.

Changed State: Released
